bio/prinseq-plus-plus/wrapper.py

```
@@ -0,0 +1,73 @@
+    raise ValueError("Invalid output file format")
+
+
+# Run Prinseq++
+with tempfile.TemporaryDirectory() as tmpdir:
+    out_prefix = "out"
+
+    shell(
+        "prinseq++ -threads {snakemake.threads} -fastq {snakemake.input[0]} {file2} {extra} -out_name {tmpdir}/{out_prefix} {log}"
+    )
+
+    if out := snakemake.output.get("r1"):
```

Ruff

Undefined name `snakemake`

(F821)